system/system.yaml

```diff
@@ -81,10 +81,7 @@
 
     ## Install frontend and backend ##
     - name: Copy nginx configuration file
-      copy:
-        src: '{{ source_dir }}/system/etc/nginx/sites-available/default'
-        dest: /etc/nginx/sites-available/default
-        remote_src: yes
+      template: src={{ source_dir }}/system/etc/nginx/sites-available/default.j2 dest=/etc/nginx/sites-available/default
```

Comment:
Does this src file exist? I noticed you added a `.j2` file for the backend service, but not for the nginx one?

Reply:
I forgot about this one. I had it on my master fork.